app/pages/about/team-page.cjsx

```diff
@@ -223,19 +348,45 @@ teamMembers =
     bio: counterpart "team.content.chrisLintott.bio"
     image: "/assets/team/chris-l.png"
     location: "oxford"
+  chrisSnyder:
+    name: "Chris Snyder"
+    twitter: "bumishness"
+    title: counterpart "team.content.chrisSnyder.title"
+    bio: counterpart "team.content.chrisSnyder.bio"
+    image: "/assets/team/chriss.png"
+    location: "alumni"
   christopherDoogue:
     name: "Christopher Doogue"
     title: counterpart "team.content.christopherDoogue.title"
     bio: counterpart "team.content.christopherDoogue.bio"
     image: "/assets/team/chris-d.jpg"
-    location: "oxford"
+    location: "alumni"
   colemanKrawczyk:
     name: "Coleman Krawczyk"
     twitter: "ColemanKrawczyk"
     title: counterpart "team.content.colemanKrawczyk.title"
     bio: counterpart "team.content.colemanKrawczyk.bio"
     image: "/assets/team/coleman.jpg"
     location: "portsmouth"
+  darrenMcRoy:
+    name: "Darren McRoy"
+    title: counterpart "team.content.darrenMcRoy.title"
+    bio: counterpart "team.content.darrenMcRoy.bio"
+    image: "/assets/team/darren.jpg"
+    location: "alumni"
+  darrylWright:
+    name: "Darryl Wright"
+    title: counterpart "team.content.darrylWright.title"
+    bio: counterpart "team.content.darrylWright.bio"
+    image: "/assets/team/team-member-avatar.jpg"
```

Comment:
does @dwright04 know his picture is missing? I think we can merge and update when we have a picture, just checking.